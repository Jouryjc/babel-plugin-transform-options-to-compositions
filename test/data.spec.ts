import plugin from '../lib/index'
import { transformSync } from '@babel/core'

describe('data -> reactive', () => {
  test('data -> reactive', () => {
    const { code } = transformSync(
      `
      export default {
        data () {
          return {
            lastName: 'Tom'
          }
        }
      }
    `,
      {
        presets: [
          [
            '@babel/preset-env',
            {
              modules: false,
            },
          ],
        ],
        plugins: [plugin],
      }
    )

    expect(code).toMatchSnapshot('data')
  })
})
