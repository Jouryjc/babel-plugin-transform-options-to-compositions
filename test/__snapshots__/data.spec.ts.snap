// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data -> reactive data -> reactive: data 1`] = `
"\\"use strict\\";

const state = reactive({
  lastName: 'Tom'
});"
`;
